plugins/BEdita/Core/config/Migrations/20171110095938_AddCoreEnabledCreatedModified.php

```diff
@@ -0,0 +1,96 @@
+<?php
+use Migrations\AbstractMigration;
+
+/**
+ * Add `core_type`, `created`, `modified`, `enabled` columns to `object_types`.
+ *
+ * @see https://github.com/bedita/bedita/1366
+ */
+class AddCoreEnabledCreatedModified extends AbstractMigration
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function up()
+    {
+        $this->table('object_types')
+            ->addColumn('created', 'datetime', [
+                'comment' => 'creation date',
+                'default' => null,
+                'limit' => null,
+                'null' => true,
+            ])
+            ->addColumn('modified', 'datetime', [
+                'comment' => 'last modification date',
+                'default' => null,
+                'limit' => null,
+                'null' => true,
+            ])
+            ->addColumn('core_type', 'boolean', [
+                'comment' => 'core object type flag',
+                'default' => false,
+                'length' => null,
+                'null' => false,
+            ])
+            ->addColumn('enabled', 'boolean', [
+                'comment' => 'object type active flag',
+                'default' => true,
+                'length' => null,
+                'null' => false,
+            ])
+            ->addIndex(
+                [
+                    'core_type',
+                ],
+                [
+                    'name' => 'objecttypes_coretype_idx',
+                ]
+            )
+            ->addIndex(
+                [
+                    'enabled',
+                ],
+                [
+                    'name' => 'objecttypes_enabled_idx',
+                ]
+            )
+            ->update();
+
+        // UPDATE created, modified
+        $this->query(sprintf("UPDATE object_types SET created = '%s', modified = '%s'", date('Y-m-d H:i:s'), date('Y-m-d H:i:s')));
```

Comment:
Could this be achieved by setting `created` and `modified` column with `CURRENT_TIMESTAMP` as their default value?

Reply:
done

Reply:
tried, but somehow unit tests are failing on sqlite on `BEditaShell` - mysteriously as usual :wink: